.circleci/config.yml

```diff
@@ -0,0 +1,49 @@
+version: 2
```

Comment:
I'd use this version instead https://github.com/verdaccio/verdaccio-google-cloud/blob/master/.circleci/config.yml#L123 to publish from circleCI